Automattic/wp-calypso
# Gutenboarding: Remove the ugly domains pre-purchase modal

This PR removes the ugly modal warning the user their site will be created after they pick a paid domain in gutenboarding.  The user should be redirected to the checkout with a personal plan in their cart.

how to test
========
0. Go to http://calypso.localhost:3000/gutenboarding
1. Enter a vertical and a site name 
2. Open the domains picker (top left) and pick a paid domain. 
3. You site shoudl be created and you should be redirected to the checkout, with a personal plan and your domain in the cart.
